spec/controllers/amr_heatmap_controller_spec.rb

```diff
@@ -1,4 +1,32 @@
 require 'rails_helper'
+require 'webmock/rspec'
+
+S3_JSON_BUCKET = "idseq-database".freeze
+S3_JSON_PREFIX = "amr/ontology/".freeze
+ONTOLOGY_BUILD_DATE = "2019-06-20".freeze
+
+OQXB_ONTOLOGY = {
+  "accession" => "3003923",
+  "label" => "oqxB",
+  "synonyms" => [
+
+  ],
+  "description" => "RND efflux pump conferring resistance to fluoroquinolone",
+  "geneFamily" => [
+    {
+      "label" => "subunit of efflux pump conferring antibiotic resistance",
+      "description" => "Subunits of efflux proteins that pump antibiotic out of a cell to confer resistance.",
+    },
+  ],
+  "drugClass" => {
+    "label" => "Fluroquinolones",
+    "description" => "The fluoroquinolones are a family of synthetic broad-spectrum antibiotics that are 4-quinolone-3-carboxylates. These compounds interact with topoisomerase II (DNA gyrase) to disrupt bacterial DNA replication, damage DNA, and cause cell death.",
```

Comment:
A lot of these long strings are duplicated twice. Can you extract them into constants?